mahjong/record/reader.py

```diff
@@ -168,6 +168,14 @@ def to_paifu(self):
         paifu_data = []
         for i in self.game_list:
             paifu_data.extend(i.to_paifu())
+        fin1 = {'event_type': 'FIN1', 'player': '0', 'score': str(self.end_score[0]) + '00'}
+        fin2 = {'event_type': 'FIN2', 'player': '1', 'score': str(self.end_score[1]) + '00'}
+        fin3 = {'event_type': 'FIN3', 'player': '2', 'score': str(self.end_score[2]) + '00'}
+        if len(self.players) == 4:
+            fin4 = {'event_type': 'FIN4', 'player': '3', 'score': str(self.end_score[3]) + '00'}
```

Comment:
`str(...) + '00'` may cause weird representation like `'000'`. Is this expected? If not consider using `self.end_score[i]*100` instead.